__init__.py

```diff
@@ -1,85 +1,128 @@
-# Copyright 2016 Mycroft AI, Inc.
+# Copyright 2017, Mycroft AI Inc.
 #
-# This file is part of Mycroft Core.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
-# Mycroft Core is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
+#    http://www.apache.org/licenses/LICENSE-2.0
 #
-# Mycroft Core is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-from mycroft.version import CORE_VERSION_MAJOR, \
-     CORE_VERSION_MINOR, CORE_VERSION_BUILD
 import datetime
-from os.path import abspath
 import tzlocal
-from adapt.intent import IntentBuilder
 from astral import Astral
 from pytz import timezone
-from mycroft.skills.core import MycroftSkill
-from mycroft.util.log import getLogger
 import time
-from threading import Timer
 
-__author__ = 'ryanleesipes', 'jdorleans', 'connorpenrod', 'michaelnguyen'
+from adapt.intent import IntentBuilder
+import mycroft.audio
+from mycroft.skills.core import MycroftSkill, intent_handler
+import mycroft.client.enclosure.display_manager as DisplayManager
+
+
+# TODO: Move to mycroft.util.format.py
+def nice_time(dt, lang="en-us", speech=True, use_24hour=False,
+              use_ampm=False):
+    '''
+    Format a time to a comfortable human format
+
+    For example, generate 'five thirty' for speech or '5:30' for
+    text display.
+
+    Args:
+        dt (datetime): date to format (assumes already in local timezone)
+        lang (str): code for the language to use
+        speech (bool): format for speech (default/True) or display (False)=Fal
+        use_24hour (bool): output in 24-hour/military or 12-hour format
+        use_ampm (bool): include the am/pm for 12-hour format
+    Returns:
+        (str): The formatted time string
+    '''
 
-LOGGER = getLogger(__name__)
+    if use_24hour:
+        # e.g. "03:01" or "14:22"
+        str = dt.strftime("%H:%M")
```

Comment:
Not really an issue but consider using `string` instead if `str` since str is the builtin string type.